src/gui.cls.php

```diff
@@ -499,7 +501,7 @@ public function frontend_shortcut() {
 			) );
 		}
 
-		if ( $this->cls( 'CSS' )->has_ccss_folder() ) {
+		if ( $this->has_cache_folder( 'ccss', $subsite_id ) ) {
```

Comment:
For what reason did you pass in `$subsite_id` while not directly generate it in the function definition? Won't it be easier as you don't need to pass each time?

Reply:
Ah I see, ok I will pass it in the has_cache_folder